packages/editor/src/components/EditorContainer.tsx

```diff
@@ -291,63 +294,30 @@ const EditorContainer = () => {
     }
     setToggleRefetchScenes(!toggleRefetchScenes)
   }
-  /*
-  const onExportProject = async () => {
-    if (!sceneName) return
-    const options = await new Promise<DefaultExportOptionsType>((resolve) => {
-      setDialogComponent(
-        <ExportProjectDialog
-          defaultOptions={Object.assign({}, DefaultExportOptions)}
-          onConfirm={resolve}
-          onCancel={resolve}
-        />
-      )
-    })
-
-    if (!options) {
-      setDialogComponent(null)
-      return
-    }
-
-    const abortController = new AbortController()
-
-    setDialogComponent(
-      <ProgressDialog
-        title={t('editor:exporting')}
-        message={t('editor:exportingMsg')}
-        cancelable={true}
-        onCancel={() => abortController.abort()}
-      />
-    )
-
-    try {
-      const { glbBlob } = await exportScene(options)
 
-      setDialogComponent(null)
-
-      const el = document.createElement('a')
-      el.download = Engine.scene.name + '.gltf'
-      el.href = URL.createObjectURL(glbBlob)
-      document.body.appendChild(el)
-      el.click()
-      document.body.removeChild(el)
-    } catch (error) {
-      if (error['aborted']) {
-        setDialogComponent(null)
-        return
+  const onImportAsset = async () => {
+    const el = document.createElement('input')
```

Comment:
make sure we are cleaning up this element in case it is not automatically garbage collected